spec/models/solidus_volume_pricing/pricer_spec.rb

```diff
@@ -0,0 +1,280 @@
+require 'spec_helper'
+
+RSpec.shared_examples 'having the variant price' do
+  it 'uses the variants price' do
+    is_expected.to eq('$10.00')
+  end
+end
+
+RSpec.describe SolidusVolumePricing::Pricer do
+  it 'inherits from default variant pricer' do
+    expect(described_class < Spree::Variant::PriceSelector).to be(true)
```

Comment:
This doesn't really provide much value as a test, but it's also completely fine.

The way we could get value from these kind of tests is by providing a shared example set to "lint" the interface of PriceSelector, as because in ruby we prefer duck typing rather that type checking.

`it_behaves_like "an implementation of a Spree::Variant::PriceSelector"` which would simply check that it responds to the public interface required by the subclass.

See how rails provides a class for verifying your "custom models" are rails compliant:

http://api.rubyonrails.org/files/activemodel/lib/active_model/lint_rb.html